BurnOutSharp/ProtectionType/Tages.cs

```diff
@@ -20,110 +20,190 @@ public string CheckContents(string file, byte[] fileContent, bool includeDebug,
             if (sections == null)
                 return null;
 
-            // Get the last section and read after it
-            var lastSection = sections.LastOrDefault();
-            if (lastSection != null)
-            {
-                int sectionAddr = (int)lastSection.PointerToRawData;
-                int sectionEnd = sectionAddr + (int)lastSection.VirtualSize;
-                var matchers = new List<ContentMatchSet>
-                {
-                    // tagesprotection.com
-                    new ContentMatchSet(
-                        new ContentMatch(new byte?[]
-                        {
-                            0x74, 0x61, 0x67, 0x65, 0x73, 0x70, 0x72, 0x6F,
-                            0x74, 0x65, 0x63, 0x74, 0x69, 0x6F, 0x6E, 0x2E,
-                            0x63, 0x6F, 0x6D
-                        }, start: sectionEnd),
-                    Utilities.GetFileVersion, "TAGES"),
-                };
-
-                string match = MatchUtil.GetFirstMatch(file, fileContent, matchers, includeDebug);
-                if (!string.IsNullOrWhiteSpace(match))
-                    return match;
-            }
+            /* 
+                Expected files to contain "TagesSetup"/"Application TagesSetup":
+                DrvSetup.exe
+                DrvSetup_x64.exe
+                TagesSetup.exe
+                TagesSetup_x64.exe
+             
+                Expected files to contain "Tages activation client"/"T@GES":
+                TagesClient.exe
+                (There is generally a TagesClient.dat accompanying the TagesClient.exe.)
+            */
+            string name = Utilities.GetFileDescription(pex);
+            if (!string.IsNullOrWhiteSpace(name) && name.StartsWith("TagesSetup", StringComparison.OrdinalIgnoreCase))
+                return $"TAGES Driver Setup {GetVersion(pex)}";
+            else if (!string.IsNullOrWhiteSpace(name) && name.StartsWith("Tagès activation client", StringComparison.OrdinalIgnoreCase))
+                return $"TAGES Activation Client {GetVersion(pex)}";
+
+            name = Utilities.GetProductName(pex);
+            if (!string.IsNullOrWhiteSpace(name) && name.StartsWith("Application TagesSetup", StringComparison.OrdinalIgnoreCase))
+                return $"TAGES Driver Setup {GetVersion(pex)}";
+            else if (!string.IsNullOrWhiteSpace(name) && name.StartsWith("T@GES", StringComparison.OrdinalIgnoreCase))
+                return $"TAGES Activation Client {GetVersion(pex)}";
 
             // TODO: Obtain a sample to find where this string is in a typical executable
-            if (includeDebug)
+            var contentMatchSets = new List<ContentMatchSet>
```

Comment:
If you're adding the `[DEBUG]` string here, keep this inside of the `if (includeDebug)` brackets.